app/models/concerns/acts_as_profile_field.rb

```diff
@@ -0,0 +1,16 @@
+concern :ActsAsProfileField do
```

Comment:
This is a [shortcut method](http://api.rubyonrails.org/classes/Module/Concerning.html#method-i-concerning) defined in `Module::Concerning` which is equivalent to the following:

```ruby
module ActsAsProfileField
  extend ActiveSupport::Concern

   ...
end
```